python/.gitignore

```diff
@@ -3,6 +3,5 @@ build
 neuroglancer.egg-info
 _neuroglancer.so
 _mesher.so
-ext/src/endian.h
```

Comment:
Why would we ignore this file? If someone changes this we want them to add the modifications here